streaming/vision/convert/README.md

```diff
@@ -0,0 +1,119 @@
+# Dataset preparation
+
+To use Streaming Dataset, We first convert the dataset from its native format to MosaicML's streaming dataset format (a collection of binary `.mds` files). Once in `.mds` format, we can store the dataset in a central location (filesystem, S3, GCS, or OCI) and stream the data to any compute cluster, with any number of devices, and any number of CPU workers, and it all just works.
+
+Follow the below steps to convert the Computer Vision dataset into a streaming MDS format. Also checkout the supported [MDSWriter()](https://streaming.docs.mosaicml.com/en/latest/api_reference/generated/streaming.MDSWriter.html) parameters for advanced usage.
+
+## 1. [ADE20K](https://groups.csail.mit.edu/vision/datasets/ADE20K/)
+
+**Instructions:**
+
+1. Download the ADE20K dataset from [here](https://groups.csail.mit.edu/vision/datasets/ADE20K/).
+2. That will result in this directory structure:
+    <!--pytest.mark.skip-->
+    ```bash
+    ├── annotations
+    │   ├── training
+    │   └── validation
+    └── images
+        ├── training
+        └── validation
+    ```
+
+3. Run the [ade20k.py](ade20k.py) script as shown below with the minimalist arguments. The script converts the `train` and `val` dataset split into their own directory. For advanced users, you can look at the supported arguments for [ade20k.py](ade20k.py) and change according to your own needs.
+      <!--pytest.mark.skip-->
+      ```
+      python ade20k.py --in_root <Above directory> --out_root <local or remote directory path to save output MDS shard files>
+      ```
+
+## 2. [CIFAR10](https://www.cs.toronto.edu/~kriz/cifar.html)
+
+**Instructions:**
+
+1. Run the [cifar10.py](cifar10.py) script as shown below with the minimalist arguments which also downloads the raw CIFAR10 dataset if it doesn't exist. For advanced users, you can look at the supported arguments for [cifar10.py](cifar10.py) and change according to your own needs.
```

Comment:
```suggestion
1. Run the [cifar10.py](cifar10.py) script as shown below with the minimalist arguments. The CIFAR10 dataset will be automatically downloaded if it doesn't exist locally. For advanced use cases, please see the supported arguments for [cifar10.py](cifar10.py) and modify as necessary.
```

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/183/commits/4d242aaf5f26f73278a555a3021da6e328d0a28b.